tests/checkpoint/test_checkpoint.py

```diff
@@ -254,18 +254,14 @@ def test_basic_checkpoint_config_validation(
     )
     captured = capsys.readouterr()
     assert any(
```

Comment:
@Kilo59 Out of curiosity, is there anything (no pun intended) with `any()`?  Thanks.

Reply:
https://beta.ruff.rs/docs/rules/unnecessary-comprehension-any-all/